ts/components/session/SessionSearchInput.tsx

```diff
@@ -42,4 +75,28 @@ export class SessionSearchInput extends React.Component<Props> {
       }
     }
   }
+
+  public handleUndo() {
+    document.execCommand('undo');
+  }
+
+  public handleRedo() {
+    document.execCommand('redo');
+  }
+
+  public handleCut() {
+    document.execCommand('cut');
+  }
+
+  public handleCopy() {
+    document.execCommand('copy');
+  }
+
+  public handlePast() {
+    document.execCommand('paste');
+  }
+
+  public handleSelectAll() {
+    document.execCommand('selectAll');
+  }
 }
```

Comment:
That a lot wordy to add all those calls here.
Can you call them directly on the handler just like that: 
```
          <MenuItem onClick={() => document.execCommand('selectAll')}>
            {window.i18n('editMenuSelectAll')}
          </MenuItem>
```

If yes, you can remove all the added .bind calls in the constructor too